themes/bootstrap3/templates/search/pagination-item.phtml

```diff
@@ -1,11 +1,16 @@
 <?php
   $classes = array_merge(['page-item'], $this->liClasses ?? []);
+  $aAttrs = $this->htmlAttributes([
+    'class' => 'page-link',
+    'aria-label' => $this->label,
+  ]);
   if ($this->page == $this->current) {
-      $classes[] = 'active';
+    $classes[] = 'active';
+    $aAttrs['aria-current'] = 'page';
   }
 ?>
 <li class="<?=implode(' ', $classes)?>">
-  <a class="page-link" href="<?=$this->results->getUrlQuery()->setPage($this->page)?>" aria-label="<?=$this->escapeHtmlAttr($this->label)?>">
+  <a<?=$aAttrs?> href="<?=$this->results->getUrlQuery()->setPage($this->page)?>">
```

Comment:
Seems a little strange to use the attribute helper for everything except href. I'm guessing you're doing this because of the escaping behavior of the URL Query Helper, but you could just do `$this->results->getUrlQuery()->setPage($this->page)->getParams(false)` to get an unescaped version, if you wanted to put everything in the attribute helper.

Reply:
Right, fixed.